modules/phase_field/tests/initial_conditions/TricrystalTripleJunctionIC.i

```diff
@@ -0,0 +1,121 @@
+  ny = 12 # Number of elements in the y direction
+  xmax = 1000 # Maximum x-coordinate of mesh
+  xmin = 0 # Minimum x-coordinate of mesh
+  ymax = 1000 # Maximum y-coordinate of mesh
+  ymin = 0 # Minimum y-coordinate of mesh
+  elem_type = QUAD4 # Type of elements used in the mesh
+  uniform_refine = 3
+[]
+
+[GlobalParams]
+  # Parameters used by several kernels that are defined globally to simplify input file
+  op_num = 3 # Number of order parameters used
+  var_name_base = gr # base name of grains
+  v = 'gr0 gr1 gr2' # Names of the grains
+  theta1 = 135 # Angle the first grain makes at the triple junction
+  theta2 = 135 # Angle the second grain makes at the triple junction
+  length_scale = 1.0e-9 # Length scale in nm
+  time_scale = 1.0e-9 # Time scale in ns
+[]
+
+[Variables]
+  [./gr0]
+    order = FIRST
+    family = LAGRANGE
+    [./InitialCondition]
+       type = TricrystalTripleJunctionIC
+       op_index = 1
+    [../]
+  [../]
+
+  [./gr1]
+    order = FIRST
+    family = LAGRANGE
+    [./InitialCondition]
+       type = TricrystalTripleJunctionIC
+       op_index = 2
+    [../]
+  [../]
+
+  [./gr2]
+    order = FIRST
+    family = LAGRANGE
+    [./InitialCondition]
+       type = TricrystalTripleJunctionIC
+       op_index = 3
+    [../]
+  [../]
+[]
+
+[AuxVariables]
+  [./bnds]
+    # Variable used to visualize the grain boundaries in the simulation
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Kernels]
+  # Kernels block where the kernels defining the residual equations are set up
+  [./PolycrystalKernel]
+    # Custom action creating all necessary kernels for grain growth.  All input parameters are up in GlobalParams
+  [../]
+[]
+
+[AuxKernels]
+  # AuxKernel block, defining the equations used to calculate the auxvars
+  [./bnds_aux]
+    # AuxKernel that calculates the GB term
+    type = BndsCalcAux
+    variable = bnds
+    execute_on = 'initial timestep_end'
+  [../]
+[]
+
+[Materials]
+  [./material]
+    # Material properties
+    type = GBEvolution
+    block = 0 # Block ID (only one block in this problem)
+    T = 450 # Constant temperature of the simulation (for mobility calculation)
+    wGB = 14 # Width of the diffuse GB
+    GBmob0 = 2.5e-6 #m^4(Js) for copper from Schoenfelder1997
+    Q = 0.23 #eV for copper from Schoenfelder1997
+    GBenergy = 0.708 #J/m^2 from Schoenfelder1997
+  [../]
+[]
+
+[Postprocessors]
+  # Scalar postprocessors
+  [./grain_tracker]
+    type = FauxGrainTracker
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  scheme = bdf2
+  solve_type = 'PJFNK'
+
+  petsc_options_iname = '-pc_type -pc_hypre_type -ksp_gmres_restart -mat_mffd_type'
+  petsc_options_value = 'hypre boomeramg 31 ds'
+  l_tol = 1e-6 # Relative tolerance for linear solves
+  nl_rel_tol = 1e-10 # Relative tolerance for non-linear solves
+  num_steps = 1
+[]
+
+[Outputs]
+  exodus = true # Outputs to the Exodus file format
+  execute_on = 'final'
+[]
+
+[Problem]
+  solve = false
```

Comment:
Haha, yeah, simplify the Executioner block down to 

```
[Executioner]
  type = Transient
[]
```

there is no solve. Our test files should be as simple as possible.